src/App.tsx

```diff
@@ -1,11 +1,63 @@
-import React from 'react';
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
 
-import './App.scss';
+import {
+  Navigate, NavLink, Route, Routes, useLocation,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { PeoplePage } from './components/PeoplePage';
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
+type Status = { isActive: boolean };
+
+const getActiveClasses = (status: Status) => classNames(
+  'navbar-item',
+  { 'has-background-grey-lighter': status.isActive },
 );
 
-export default App;
+export const App: React.FC = () => {
+  const location = useLocation();
+
+  return (
+    <>
+      <nav
+        className="navbar is-fixed-top has-background-light"
+        data-cy="nav"
```

Comment:
Nice job! But let's make some improvements! I bet this navigation can be another component